jerry-ext/external/external-gc.c

```diff
@@ -0,0 +1,31 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript-ext/external.h"
+
+/**
+ * Expose garbage collector to scripts.
+ *
+ * @return undefined.
+ */
+jerry_value_t
+jerryx_external_gc (const jerry_value_t func_obj_val __attribute__((unused)), /**< function object */
```

Comment:
We don't have access to that here, but I'll rewrite it to (void) (func_obj_val);